components/apimgt/org.wso2.carbon.apimgt.gateway/pom.xml

```diff
@@ -82,13 +82,22 @@
         </dependency>
         <dependency>
              <groupId>org.wso2.carbon</groupId>
-             <artifactId>org.wso2.carbon.utils</artifactId>       			 
+             <artifactId>org.wso2.carbon.utils</artifactId>
          </dependency>
         <dependency>
             <groupId>org.wso2.carbon</groupId>
             <artifactId>org.wso2.carbon.identity.oauth.stub</artifactId>
             <version>4.2.3</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.axis2</groupId>
+            <artifactId>axis2-kernel</artifactId>
+            <version>1.6.2</version>
```

Comment:
Versions have to go into the root pom. Please put the entire dependency under dependencyManagement section and refer it from there.